src/FOMObot/Helpers/Redis.hs

```diff
@@ -0,0 +1,51 @@
+module FOMObot.Helpers.Redis
```

Comment:
Is this representative of the behavior this module provides? E.g. if the underlying storage mechanism changed to JSON and the function names didn't need to change, `Redis` would be misrepresentative.

Seems like there's two separate aspects - preference management and DM channel checking. Might make sense to pull those apart and rename the module?